src/core/ApolloClient.ts

```diff
@@ -339,10 +384,10 @@ export class ApolloClient<TCacheShape> implements DataProxy {
 
   /**
    * This resolves a single query according to the options specified and
-   * returns a {@link Promise} which is either resolved with the resulting data
+   * returns a Promise which is either resolved with the resulting data
```

Comment:
```suggestion
   * returns a `Promise` which is either resolved with the resulting data
```

Make this an inline code block perhaps?